app/services/hyrax/collection_types/create_service.rb

```diff
@@ -37,19 +71,23 @@ class CreateService
       # @option options [True | False] :nestable if true, collections of this type can be nested
       # @option options [True | False] :discoverable if true, collections of this type can be marked Public and found in search results
       # @option options [True | False] :sharable if true, collections of this type can have participants added for :manage, :deposit, or :view access
+      # @option options [True | False] :share_applies_to_collection if true, share participant permissions are applied to the collection
+      # @option options [True | False] :share_applies_to_new_works if true, share participant permissions are applied to new works created in the collection
       # @option options [True | False] :allow_multiple_membership if true, works can be members of multiple collections of this type
       # @option options [True | False] :require_membership if true, all works must belong to at least one collection of this type.  When combined
       #   with allow_multiple_membership=false, works can belong to one and only one collection of this type.
       # @option options [True | False] :assigns_workflow if true, collections of this type can be used to assign a workflow to a work
       # @option options [True | False] :assigns_visibility if true, collections of this type can be used to assign initial visibility to a work
       # @return [Hyrax::CollectionType] the newly created collection type instance
-      def self.create_collection_type(machine_id: DEFAULT_MACHINE_ID, title: DEFAULT_TITLE, options: {})
+      def self.create_collection_type(machine_id:, title:, options: {}) # rubocop:disable Metrics/MethodLength
         opts = DEFAULT_OPTIONS.merge(options)
         ct = Hyrax::CollectionType.create!(machine_id: machine_id, title: title) do |c|
```

Comment:
Is there a reason we don't just `Hyrax::CollectionType.create!(opts.merge(machine_id: machine_id, title: title))` and do away with the block?